### PR TITLE
add middleware exceptions

### PR DESCRIPTION
**Pull request by MauricioAliendre182**
Middleware Exception Added

### CODE REVIEW

File: spring-boot-hello-world/src/main/java/com/jalasoft/convert/middleware/AudioControllerMiddleware.java [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 package com.jalasoft.convert.middleware;
 
-import com.jalasoft.convert.common.exception.FileNotFoundException;
+import com.jalasoft.convert.common.exception.MiddlewareException;
 import com.jalasoft.convert.common.logger.At18Logger;
 import javax.servlet.*;
```
Comment (HugoS922) [inline spring-boot-hello-world/src/main/java/com/jalasoft/convert/middleware/AudioControllerMiddleware.java:14]: Hello Mauricio there are still Imports with "*", please try to remove them with the correct imports

File: spring-boot-hello-world/src/main/java/com/jalasoft/convert/middleware/DocControllerMiddleware.java [hunk 2]
```diff
@@ -9,7 +9,7 @@
 
 package com.jalasoft.convert.middleware;
 
-import com.jalasoft.convert.common.exception.FileNotFoundException;
+import com.jalasoft.convert.common.exception.MiddlewareException;
 import com.jalasoft.convert.common.logger.At18Logger;
 import javax.servlet.*;
 import javax.servlet.FilterChain;
```
Comment (HugoS922) [inline spring-boot-hello-world/src/main/java/com/jalasoft/convert/middleware/DocControllerMiddleware.java:15]: Same here Mauricio

File: Download/Traducido-File.txt [hunk 3]
```diff
@@ -0,0 +1,202 @@
+﻿
```
Comment (HugoS922) [inline Download/Traducido-File.txt:1]: Hello Mauricio this is an unnecesary file, for the PR you should remove it, please try to revove it before merging

**Review by HugoS922 — CHANGES_REQUESTED** [review]
Hello @MauricioAliendre182 there are at least 6 imports with "*", please try to remove them, and the TXT file that is not used

**Comment by MauricioAliendre182** [comment]
> Hello @MauricioAliendre182 there are at least 6 imports with "*", please try to remove them, and the TXT file that is not used

Ok Hugo, I will do it, I will upload middleware exceptions in this PR too

**Pull request closed without merging**